packages/ledger-ethereum/src/EthereumLedger.ts

```diff
@@ -61,8 +61,13 @@ export default class EthereumLedger implements IBlockchain {
 
   public async initialize(): Promise<void> {
     // Set primary address
-    const [from] = await utils.getAccounts(this.web3);
+    let [from] = await utils.getAccounts(this.web3);
```

Comment:
since this takes web3, probably it's better to handle getting the address in here.